src/backend/main.ts

```diff
@@ -810,6 +811,13 @@ ipcMain.handle('checkGameUpdates', async (): Promise<string[]> => {
     oldGames = [...oldGames, ...gamesToUpdate]
   }
 
+  sendGameUpdatesNotifications().catch((e) =>
+    logError(
+      `Something went wrong sending update notifications: ${e}`,
+      LogPrefix.Backend
+    )
+  )
```

Comment:
this will run on every refresh click. shouldn't we only run this once on startup? we can add a module level flag in the `src/backend/storeManagers/index.ts` module for this

Reply:
fixed this